packages/viewer/tests/cypress/tests-e2e/3d/transitionTo3d.cy.ts

```diff
@@ -5,6 +5,8 @@ import { WGS84 } from '@swissgeo/coordinates'
 import { Math as CesiumMath } from 'cesium'
 import proj4 from 'proj4'
 
+
+// @ts-expect-error map config not a typescript file
```

Comment:
should be quite easy to transform to TS though (no imports, only const declaration)